packages/shorebird_cli/lib/src/common_arguments.dart

```diff
@@ -0,0 +1,6 @@
+/// A class that houses the name of arguments that are shared between different
+/// commands and layers.
+class CommonArguments {
+  static const String publicKeyArgName = 'public-key-path';
+  static const String privateKeyArgName = 'private-key-path';
+}
```

Comment:
I like! I wonder how much further we'll be able to build this out (for example, can we also include descriptions? Accept an arg parser and add standard options to it?)

Not for this PR, just brainstorming.

Reply:
minor but might be nice to have a `static const Options options` on the `CommandRunner` 🤷

Reply:
Yeah, I like the idea of making this more robust. I don't it would be that hard, would just take a bit of time to refactor the arguments